### ISSUE TITLE
Update loss function as its using a Mean Squared Error

### ISSUE THREAD

**Issue by raghibhaque** [raghibhaque/neuralnetwork#1]
When i was learning about this project, i decided to use the MSE error but binary cross entropy seems to be better suited at probability predictions, which suits the project better
